### PR TITLE
edit readme, delete ChangeLog and License files, edit package.json

### PR DESCRIPTION
**Pull request by Fraih001**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,64 +1 @@
-# Links
-- Slack group: https://app.slack.com/client/T024FPYBQ/C03BUFQB50V/T2VHPBHA5
-- Figma: https://www.figma.com/file/Q7LqTYZxPqUVjD8tF7ihmy/FSA-Capstone
-- Todo Board: https://github.com/orgs/Team5-Capstone/projects/1/views/1
-
```
Comment (bob-zs) [inline README.md:5]: This url doesn't go anywhere, and `orgs/testbrew` is `testbrew/testbrew`
https://github.com/orgs/testbrew/testbrew 

try changing it to

[TestBrew Repo](https://github.com/testbrew/testbrew)
with a link

**Review by bob-zs — COMMENTED** [review]
Thanks, great job cleaning all that, just 1 nit

File: README.md (hunk 1 above)
Comment (Fraih001) [reply README.md:5]: Nice, last commit should fix that.

File: README.md [hunk 2]
```diff
@@ -1 +1 @@
-https://github.com/orgs/testbrew/testbrew
+[TestBrew Repo](https://github.com/orgs/testbrew/testbrew)
```
Comment (bob-zs) [inline README.md:1]: https://github.com/orgs/testbrew/testbrew  -> https://github.com/testbrew/testbrew/

Comment (bob-zs) [reply README.md:1]: it's not an org account.

Comment (Fraih001) [reply README.md]: lol sorry, that should do it.

**Review by bob-zs — APPROVED** [review]
🎉 🎉 🎉 🎉 🎉 🎉

**Pull request merged**